src/pages/Analytics/Program/helper.js

```diff
@@ -1,4 +1,32 @@
 import mapValues from 'lodash/mapValues'
+import { validateObjectByProperty } from '../../../utils/validators'
+
+export const createInitialValues = initialValues => ({
+    id: initialValues.id || '',
+    program: initialValues.program || '',
+    visualization: initialValues.visualization || '',
+    name: initialValues.name || '',
+    group: initialValues.group || {
+        name: '',
+        id: '',
+    },
+})
+
+export const validMandatoryFields = settings => {
```

Comment:
Should this function be called `invalidMandatoryFields` instead? Seems like it's false when `settings.program` and `settings.visualization` are valid